README.md

```diff
@@ -1,41 +1,63 @@
 * accuracy & median F1-score
 * average semantic similarity between predicted and reference labels
+* combined score: combination of the above metrics expressed as a single number. This score is then used as a 'weight' for labels predicted by corresponding tool in production scenario. 
 
-For a subset of statistics, empirical distribution function can be estimated and then used to determine the p-value of obtained scores.  
-
-A common output format across tools is assumed - a tab-separated table with 2 columns: `cell_id` (or barcode) and `predicted_label`. In some use cases, `score` column might be required. Outputs from multiple tools must be stored in a single directory, with file names prefixed by tool name, e.g. `toolX_output.tsv`. A reference table is required, with the following compulsory columns: cell ids, reference labels, and corresponding cell ontology terms.
+For a subset of statistics, empirical distribution function can be estimated and then used to determine the p-values of computed scores.  
 
-Predicted labels for specific cells can be aggregated and analysed, providing evidence in favour of each label based on the overall performance of corresponding tools.
+A common output format across tools is assumed - a tab-separated table with 3 columns: cell id (or barcode), predicted labels, and corresponding prediction scores (e.g. p-values or distance metrics). In cases when scores cannot be retrieved, this column can be omitted. Outputs from multiple tools must be stored in a single directory, with file names prefixed by tool name, e.g. `toolX_output.tsv`. A reference table is required, with the following compulsory columns: cell ids, reference labels, and corresponding cell ontology terms.
 
-Example output tables can be found [here](https://github.com/ebi-gene-expression-group/cell-types-analysis/tree/master/example_output).
+As a result, a table with a set of metrics per each tool is produced. 
 
-### Installation 
-The package is installed via EBI Gene Expression Group conda channel:
+### Production scenario (novel data)
+In production scenario, we are interested in getting as accurate predictions for novel data as possible. To do so, we run a host of pre-trained classifiers against an incoming data set. Then, we aggregate predictions on a single tool basis and select top ones based on corresponding scores. This is followed by a second filtering step where predictions across different tools are analysed for consistency and semantic similarity. 
 
-```conda install -c ebi-gene-expression-group cell-types-analysis``` 
-
-Use `run_post_install_tests.sh` script to make sure installation was successful. 
+Example output tables can be found [here](https://github.com/ebi-gene-expression-group/cell-types-analysis/tree/master/example_output).
 
-Run `<script_name>.R --help` to see documentation.  
+## Commands 
+**build_cell_ontology_dict.R**: create a mapping between labels and ontology terms
+```
+build_cell_ontology_dict.R\
```

Comment:
External users might need some help with what an SDRF file is, and what a condensed SDRF file is- could you help them out? You could point to MAGE-TAB documentation for the former (if it exists- check with @anjaf ), and just describe the latter.

Reply:
Official MAGE-TAB documentation is here: http://fged.org/projects/mage-tab/

Reply:
@anjaf, thanks!